src/Inkluzitron/Extensions/BotDatabaseContextExtensions.cs

```diff
@@ -81,5 +85,29 @@ static public async Task UpdateCommandConsentAsync(this BotDatabaseContext conte
                 await context.SaveChangesAsync();
             });
         }
+
+        static public Task<DailyUserActivity> GetTodayUserActivityAsync(this BotDatabaseContext context, IUser user)
+            => GetUserActivityAsync(context, user, DateTime.Now);
```

Comment:
Non-UTC timestampy jsou cesta do pekla. Beru že SQLite/EF to neumí řešit moc dobře, ale snad bych tu radši rovnou ukládal ten string, než pracoval s DateTime.